### ISSUE TITLE
Absolute data paths broken in windows

### ISSUE THREAD

**Issue by tancik** [nerfstudio-project/nerfstudio#1293]
Trying to run `ns-train` with an absolute path fails to run.

**Comment by MvWouden** [comment]
I don't think this is still an issue, I have used absolute paths more than once without any troubles on Windows. e.g., `ns-train nerfacto --data C:\path\to\folder ...`.
